### PR TITLE
Docs content: Link

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
Updates docs content for Link.

Includes a small typo fix for Tooltip.

### CODE REVIEW

File: packages/docs/components/link.md [hunk 1]
```diff
@@ -25,148 +22,121 @@ links:
...
-If a direct email link is required, display the whole address (e.g. lauren.ipsum@okta.com). Avoid colloquial text that might obfuscate the presence of a mailto link (e.g., "Contact Us").
+- The destination of the link aids in the completion of a task on the current tab (e.g. help documentation)
+- Opening the link in the current tab would result in loss of data or task interruption (e.g. completing a long form)
 
 </Description>
 
 <Visual>
   <template>
-    <a href="mailto:donuts@okta.com">donuts@okta.com</a>
+    <a href="https://en.wikipedia.org/wiki/Cosmic_latte" target="_blank" rel="noopener">https://en.wikipedia.org/wiki/Cosmic_latte</a>
```
Comment (arnoldsandoval-okta) [inline packages/docs/components/link.md:44]: I had never heard of this 😂

File: packages/docs/components/link.md [hunk 2]
```diff
@@ -25,148 +22,121 @@ links:
...
-    <a href="#">View all reports</a>
-  </template>
-</Visual>
+</Description>
 
-<Visual variant="negative">
+<Visual>
   <template>
-    <a href="#">View</a>
+    <a href="mailto:odysseus@okta.com">odysseus@okta.com</a>
```
Comment (arnoldsandoval-okta) [inline packages/docs/components/link.md:120]: just curious: is this a real email address?

**Review by arnoldsandoval-okta — APPROVED** [review]
LGTM!

File: packages/docs/components/link.md (hunk 2 above)
Comment (edburyenegren-okta) [reply packages/docs/components/link.md:120]: it's not - but maybe it should be?

**Pull request merged**